docs/design-proposals/global-scheduler/GlobalSchedulerArchitectureDesignSpecificationFirstDraft.md

```diff
@@ -0,0 +1,416 @@
+- [1. Introduction](#1-introduction)
+  - [1.1 Problem Statement](#11-problem-statement)
+  - [1.2 Goals and Scope](#12-goals-and-scope)
+  - [1.3 Design Considerations](#13-design-considerations)
+    - [1.3.1 Monolithic Scheduler (Kubernetes, Openstack)](#131-monolithic-scheduler-kubernetes-openstack)
+    - [1.3.2 Siloed Multi-Scheduler](#132-siloed-multi-scheduler)
+    - [1.3.3 Shared-State Multi-Scheduler](#133-shared-state-multi-scheduler)
+    - [1.3.4 Siloed Multi-Scheduler with Intercommunication](#134-siloed-multi-scheduler-with-intercommunication)
+    - [1.3.5 Siloed Multi-Scheduler with Arbitrator](#135-siloed-multi-scheduler-with-arbitrator)
+    - [1.3.6 Two-Level Scheduler](#136-two-level-scheduler)
+    - [1.3.7 Summary of Scheduler Design Pain Points](#137-summary-of-scheduler-design-pain-points)
+- [2. Global Scheduler Overall Architecture Design](#2-global-scheduler-overall-architecture-design)
+  - [2.1 Key Requirement Analysis](#21-key-requirement-analysis)
+  - [2.2 Key Points of this New Scheduler Architecture Design](#22-key-points-of-this-new-scheduler-architecture-design)
+  - [2.3 High-Level Architecture Design](#23-high-level-architecture-design)
+  - [2.4 Architecture Component Description](#24-architecture-component-description)
```

Comment:
Migration:This interaction need cluster scheduler have the capability to accept scheduler request but not do the create VM task.